docker-compose-dev.yml

```
@@ -2,8 +2,8 @@ version: '3'
 
 services:
   nginx:
-    image: nginx:latest
-    container_name: nginx
+    container_name: nginx_dev
```

다른 컨테이너들과 통일시키는 게 좋을 것 같습니다
```suggestion
    container_name: the-pong_nginx_dev
```